webextensions/api/privacy.json

```
@@ -119,7 +119,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "≤61"
+                  "version_added": "49"
```

:x: https://github.com/chromium/chromium/commit/c537ba208c24c14ca0259e0492041b3f2e6a6eb9 landed in [48.0.2557.0](https://github.com/chromium/chromium/releases/tag/48.0.2557.0) before 48.0.2564.

```suggestion
                  "version_added": "48"
```